src/components/button-bar/button-bar.spec.tsx

```diff
@@ -278,4 +279,31 @@ describe("with different button props", () => {
     );
     expect(wrapper.containsMatchingElement(<span>subtext</span>)).toBeTruthy();
   });
+
+  describe("Button Minor variant", () => {
+    let wrapper: ReactWrapper;
+
+    beforeEach(() => {
+      wrapper = mount(
+        <ButtonBar>
+          <ButtonMinor>Click me</ButtonMinor>
+          <ButtonMinor>Click me</ButtonMinor>
+          <ButtonMinor>Click me</ButtonMinor>
+        </ButtonBar>
+      );
+    });
+
+    it("changes color, background-color, and border-color on hover", () => {
+      const button = wrapper.find('[data-component="button-minor"]').first();
+      button.simulate("mouseenter");
+      assertStyleMatch(
+        {
+          color: `var(--colorsActionMinor500)`,
+          borderColor: `var(--colorsActionMinor500)`,
+        },
+        button
+      );
+      button.simulate("mouseleave");
```

Comment:
You don't need to simulate the `mouseleave` here fyi but it's not critical to remove